packages/MarkdownEditor-Core.package/MarkdownEmphasisDelimiter.class/instance/locatedBetween.and..st

```diff
@@ -0,0 +1,6 @@
+initialize-release
+locatedBetween: precedingCharacter and: followingCharacter
```

Comment:
I think this method name should state what the method does. Especially down in 'postProcessTokens', the method call is confusing. You say 'token do something' and then it's just 'token locatedBetween', which does not say what the token actually _does_ with itself. It is setting the flanking for a token, so a suggestion would be '(decide)flankingForPreceding:Following:' and then that splits into deciding left and right flanking.

Reply:
As a general token has no concept of flanking, I'd something that is not specific to flanking, like `processPreceding:following:` (or other)

Reply:
What about `setPreceding:following`? What I mean we are informing the token about its preceding and following characters so set fits it pretty well. The fact that it doesn't save them in an instance variable isn't really a problem. The set-method idiom still applies.